### PR TITLE
Update org-msg org-msg variable.

### LINKED ISSUE #4294: org-msg does not send plain text alternative due to change in variables
The `org-msg-text-plain-alternative` variable has been deprecated in org-msg.

The new way to have org-msg send html + text is to use: `org-msg-default-alternatives `

Ref: https://github.com/jeremy-compostella/org-msg/pull/71

### PR DESCRIPTION
**Pull request by ptoal**
Fixes #4294.

### CODE REVIEW

File: modules/email/mu4e/config.el [hunk 1]
```diff
@@ -110,7 +110,7 @@
   :config
   (setq org-msg-startup "inlineimages"
         org-msg-greeting-name-limit 3
-        org-msg-text-plain-alternative t))
+        org-msg-default-alternatives `(html text)))
```
Comment (hlissner) [inline modules/email/mu4e/config.el:113]: Please replace the backquote with a single quote.

```diff
-       org-msg-default-alternatives `(html text)))
+       org-msg-default-alternatives '(html text)))
```

**Review by hlissner — CHANGES_REQUESTED** [review]

**Review by hlissner — APPROVED** [review]

**Comment by ptoal** [comment]
Sorry about that.  Having a bad git day.

**Comment by hlissner** [comment]
No worries! Thanks for the help!

**Comment by tecosaur** [comment]
Why do people keep messing with my PR :cry: 
I don't like dealing with conflicts

**Comment by ptoal** [comment]
I'm sorry.  I don't understand the issue?  I found problem that was caused by upstream change in org-msg, and fixed it.  No idea what you mean about your PR?

**Comment by tecosaur** [comment]
Oh, don't mind me that much - I just have a big mu4e PR that has a bunch of stuff like this already taken care of, see: https://github.com/hlissner/doom-emacs/pull/3971

**Comment by arvindsv** [comment]
I'm not sure if this is the right place to mention this or if I should create a new issue, but there is a difference between:

```
org-msg-default-alternatives '(html text)
```

and

```
org-msg-default-alternatives '(text html)
```

In section 7.2.3 of , [RFC-1341](https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html) it says:

```
In general, user agents that compose multipart/alternative entities should place the body parts in increasing order of preference, that is, with the preferred format last. For fancy text, the sending user agent should put the plainest format first and the richest format last. Receiving user agents should pick and display the last format they are capable of displaying. In the case where one of the alternatives is itself of type "multipart" and contains unrecognized sub-parts, the user agent may choose either to show that alternative, an earlier alternative, or both. 
```

Since the default value was set to `'(html text)`, this caused my emails sent to favour `text over `html` in the display. Especially in GMail, it caused all formatting to be lost. I had to change it to `'(text html)` to get it to show properly.

I'm not sure what the earlier behaviour was, when `org-msg-text-plain-alternative` was set to `t`.

**Pull request merged**